jobs/config.json

```diff
@@ -2491,6 +2491,26 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gce-master-new-downgrade-cluster": {
+    "args": [
+      "--check-leaked-resources",
+      "--check-version-skew=false",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gce-new-master-upgrade-cluster.env",
+      "--extract=ci/latest-1.7",
+      "--extract=ci/latest",
```

Comment:
So these flag orders matter? and the -skew flag kind of reverses the way in which it matters?

Reply:
Yes the order of two --extract matters. I think --skew is to reverse which e2e binary to run.